charts/ingress-nginx/templates/admission-webhooks/job-patch/job-createSecret.yaml

```diff
@@ -33,7 +33,7 @@ spec:
       containers:
         - name: create
           {{- with .Values.controller.admissionWebhooks.patch.image }}
-          image: "{{.repository}}:{{ .tag }}{{- if (.digest) -}} @{{.digest}} {{- end -}}"
+          image: "{{- if.registry -}}{{ .registry }}/{{- end -}}{{.repository}}:{{ .tag }}{{- if (.digest) -}} @{{.digest}} {{- end -}}"
```

Comment:
missing space between "if" and ".registry"

Reply:
thank you again, for the feedback @amirschw for the feedback and help. I incorporated your feedback.